src/ConnectedMode/Migration/ConnectedModeMigration.cs

```diff
@@ -141,10 +145,22 @@ private async Task MigrateImplAsync(BoundSonarQubeProject oldBinding, IProgress<
             // Trigger a re-fetch of suppressions so the Roslyn settings are updated.
             await suppressionIssueStoreUpdater.UpdateAllServerSuppressionsAsync();
 
+            if (shareBinding)
+            {
+                progress?.Report(new MigrationProgress(0, 1, "Saving Shared Binding Config...", false));
+                SaveSharedBinding(oldBinding);
+            }
+
             progress?.Report(new MigrationProgress(0, 1, "Migration finished successfully!", false));
             logger.WriteLine(MigrationStrings.Process_Finished);
         }
 
+        private void SaveSharedBinding(BoundSonarQubeProject oldBinding)
```

Comment:
why is it oldBinding? it's still the up-to-date data?

Reply:
Extract Method used this because of the name of the original let me fix :D